Model/Import/Category.php

```diff
@@ -809,6 +817,18 @@ protected function reindexUpdatedCategories($categoryId)
 
             $category = $this->categoryRepository->get($categoryId, $storeId);
 
+            // Make sure 301 redirects are being generated for categories
+            $saveRewritesHistory = $this->_scopeConfig->isSetFlag(
+                \Magento\CatalogUrlRewrite\Block\UrlKeyRenderer::XML_PATH_SEO_SAVE_HISTORY,
+                \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
+                $storeId
+            );
+            $category->setData('save_rewrites_history', $saveRewritesHistory);
+            $urlRewrites = array_merge(
+                $this->categoryUrlRewriteGenerator->generate($category, true),
+                $this->urlRewriteHandler->generateProductUrlRewrites($category)
+            );
+
             $urlRewrites = $this->categoryUrlRewriteGenerator->generate($category, true);
```

Comment:
Aren't you overwriting `$urlRewrites` with this line directly? This doesn't look as it was intended.

Reply:
Indeed `$urlRewrites = $this->categoryUrlRewriteGenerator->generate($category, true);` should be removed.

(@koenner01 is a colleague of mine, and I can see in a project of ours where we've changed this code, that this line is gone indeed, so it was probably accidentally forgotten in this PR)

Reply:
@koenner01 can you fix the PR?